From 9add283b49ff65cf7209b710dc203f9e06977a0b Mon Sep 17 00:00:00 2001
From: Wang Haoran <Haoran.Wang@freescale.com>
Date: Thu, 19 Mar 2015 13:59:18 +0800
Subject: [PATCH 457/602] MA-6409 Remove mobile configurations for sabresd_6sx
 board

Modify the overlay configurations file to remove all of the
mobile(3G) configurations to pass CTS android.telephony.cts.

Signed-off-by: Wang Haoran <Haoran.Wang@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml        | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
index 25436d3..5edb7b0 100755
--- a/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
@@ -51,7 +51,6 @@
                       [# simultaneous connection types]"  -->
     <string-array translatable="false" name="radioAttributes">
         <item>"1,1"</item>
-        <item>"0,1"</item>
         <item>"7,1"</item>
         <item>"9,1"</item>
     </string-array>
@@ -64,13 +63,6 @@
     <!-- the 6th element indicates boot-time dependency-met value. -->
     <string-array translatable="false" name="networkAttributes">
         <item>"wifi,1,1,1,-1,true"</item>
-        <item>"mobile,0,0,0,-1,true"</item>
-        <item>"mobile_mms,2,0,3,60000,true"</item>
-        <item>"mobile_supl,3,0,3,60000,true"</item>
-        <item>"mobile_hipri,5,0,4,60000,true"</item>
-        <item>"mobile_fota,10,0,3,60000,true"</item>
-        <item>"mobile_ims,11,0,3,60000,true"</item>
-        <item>"mobile_cbs,12,0,3,60000,true"</item>
         <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
-- 
1.8.0

